recipe/meta.yaml

```diff
@@ -26,7 +29,8 @@ requirements:
     - python
 
 test:
-  imports:
```

Comment:
Agreed. But since the package is broken and this fixes it we can do it later. Merging this as-is for now.

PS: @snorfalorpagus and @astaric consider adding yourself as maintainers.